runtime/near-evm-runner/benches/benches/cryptozombies.rs

```diff
@@ -36,3 +38,49 @@ pub fn deploy_code(c: &mut Criterion) {
         })
     });
 }
+
+fn alice_account() -> AccountId {
+    "alice.near".to_string()
+}
+fn bob_account() -> AccountId {
+    "bob.near".to_string()
+}
+
+pub fn transfer_erc721(c: &mut Criterion) {
+    let mut ext = MockedExternal::new();
+    let bytes =
+        hex::decode(include_bytes!("../../tests/build/ZombieOwnership.bin").to_vec()).unwrap();
+    let outcome = evm_call(&mut ext, "alice", "deploy_code", bytes, false).0.unwrap();
+    let contract_id = address_from_arr(&outcome.return_data.as_value().unwrap());
+
+    let (input, _decoder) = cryptozombies::functions::create_random_zombie::call("test");
+    let args_create = encode_call_function_args(contract_id, input);
+
+    let alice_address = near_evm_runner::utils::near_account_id_to_evm_address(&alice_account());
+    let bob_address = near_evm_runner::utils::near_account_id_to_evm_address(&bob_account());
+
+    c.bench_function("transfer_erc721", |b| {
+        // first zombie ID
+        let mut zombie_id = U256::zero();
+        b.iter(|| {
+            // create zombie
+            //let mut ext_copy = ext.clone();
+            evm_call(&mut ext, "alice", "call", args_create.clone(), false).0.unwrap();
```

Comment:
create only works once per sender in the ZombieAttack contract. all the rest of creates should be failing:
https://github.com/near/nearcore/blob/master/runtime/near-evm-runner/tests/contracts/ZombieAttack.sol#L118

Reply:
> create only works once per sender in the ZombieAttack contract. all the rest of creates should be failing:
> https://github.com/near/nearcore/blob/master/runtime/near-evm-runner/tests/contracts/ZombieAttack.sol#L118

Thanks for pointing that out, I missed that this was the reason the transfers failed.


> Also next line keeps trying to transfer zombie_id == 0 from alice to bob, but after the first transfer it will be already at bob's, so should fail as well.

I think that's not correct, I was increasing the `zombie_id` below, so that it should always transfer Alice's zombies.

Given that the contract only allows one zombie per sender, do you think it's OK for performance measurement if I just clone the state in every iteration, create a zombie with ID 0 and transfer that one? That would be the easiest and would still give an upper bound on transfer time.

I could also modify the Zombie contract to allow multiple zombies per sender or create more senders (with 1 zombie each).

Reply:
I went with the first approach, cloning the state in every iteration.